src/main/java/cinema/controllers/MovieController.java

```diff
@@ -0,0 +1,41 @@
+package cinema.controllers;
+
+import cinema.mappers.MovieMapper;
+import cinema.model.Movie;
+import cinema.model.dto.MovieResponseDto;
+import cinema.service.MovieService;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class MovieController {
+    MovieService movieService;
+
+    public MovieController(MovieService movieService) {
+        this.movieService = movieService;
+    }
+
+    @PostMapping("/movies")
+    public void add(@RequestBody MovieResponseDto movieResponseDto) {
+        MovieMapper mapper = new MovieMapper();
+        mapper.setMovieResponseDto(movieResponseDto);
```

Comment:
let's create methods in mapper which will take MovieRequestDto and map it to Movie — mapToMovie() for example

Reply:
but i thougth that it's bad idea to create two identical classes MovieRequestDto and MovieResponseDto

Reply:
done

Reply:
done